src/System.IO.Pipes/src/System/IO/Pipes/NamedPipeServerStream.Unix.cs

```diff
@@ -123,12 +122,9 @@ public void Disconnect()
 
         protected override void Dispose(bool disposing)
         {
-            // If we created the FIFO object, be a good citizen and clean it up.
-            // If this doesn't happen, worst case is we leave a temp file around.
-            if (_createdFifo && _path != null)
-            {
-                Interop.libc.unlink(_path); // ignore any errors
-            }
+            // Don't clean up the FIFO object, regardless of whether we created it. If we remove
```

Comment:
I'll move the comment to where we create the FIFO and change it up a bit to make sense there.